examples/liver-scriptcontroller.py

```diff
@@ -0,0 +1,103 @@
+# Required import for python
+import Sofa
+
+
+# Choose in your script to activate or not the GUI
+USE_GUI = True
+
+
+def main():
+    import SofaRuntime
+    import Sofa.Gui
+    SofaRuntime.importPlugin("SofaOpenglVisual")
```

Comment:
No need to add them both here and within the scene (using the `RequiredPlugin` component).